tests/LINEBot/EventRequestParserTest.php

```diff
@@ -62,15 +62,24 @@ class EventRequestParserTest extends TestCase
    "message":{
     "id":"contentid",
     "type":"text",
-    "text":"message (love)",
+    "text":"@example Hello, world! (love)",
```

Comment:
Please create a new message for testing mention.

Reply:
I added new `TextMessageEvent` for testing mention.